docs/usage/supported_k8s_versions.md

```diff
@@ -1,20 +1,43 @@
 # Supported Kubernetes Versions
 
-Currently, the Gardener supports the following Kubernetes versions:
+Currently, Gardener supports the following Kubernetes versions:
 
-## Garden cluster version
+## Garden Clusters
 
-The minimum version of the garden cluster that can be used to run Gardener is **`1.20.x`**.
+The minimum version of a garden cluster that can be used to run Gardener is **`1.20.x`**.
 
-## Seed cluster versions
+## Seed Clusters
 
 The minimum version of a seed cluster that can be connected to Gardener is **`1.20.x`**.
 Please note that Gardener does not support **`1.25`** seeds yet.
 
-## Shoot cluster versions
+## Shoot Clusters
 
 Gardener itself is capable of spinning up clusters with Kubernetes versions **`1.20`** up to **`1.25`**.
 However, the concrete versions that can be used for shoot clusters depend on the installed provider extension.
 Consequently, please consult the documentation of your provider extension to see which Kubernetes versions are supported for shoot clusters.
 
 > 👨🏼‍💻 Developers note: [This document](../development/new-kubernetes-version.md) explains what needs to be done in order to add support for a new Kubernetes version.
+
+## Support Timeline
+
+The Kubernetes project maintains the most recent three minor releases and releases a new minor version every 4 months.
+This means that a release has patch support for approximately 1 year.
+See [this document](https://kubernetes.io/releases/) for the official upstream information.
+
+In the past, the Gardener project did not have a policy regarding the number of supported Kubernetes versions at the same time.
+Beginning with 2023, a new policy has been introduced:
+
+> The Gardener project supports the last four Kubernetes minor versions and drops support for the oldest minor version as soon as support for a new minor version has been introduced.
```

Comment:
Doesn't it make more sense to support only the last 3 minor versions, just like Kubernetes itself? Who should otherwise provide (security) patches?

```suggestion
> The Gardener project supports the last three Kubernetes minor versions and drops support for the oldest minor version as soon as support for a new minor version has been introduced.
```

That would fit (kind of) nicely as Gardener needs 1-2 months to add support for a new version and that's the extra maintenance given for security patches. See https://kubernetes.io/releases/patch-releases/#support-period:

> The first twelve months of this timeframe will be considered the standard period.
> During the two-month [extra] maintenance mode period, Release Managers may cut additional maintenance releases to resolve:
> - CVEs (under the advisement of the Security Response Committee)
> - dependency issues (including base image updates)
> - critical core component issues

I.e. if Gardener supports the last three minor versions, but has added support for the latest version 1-2 months after its official release, it means we will get security patches for the entire time Gardener supports these 3 minor versions (the oldest minor version will be out of non-critical functional maintenance then, but will be still within security patch maintenance because of the 2 extra months, if we never need more time than 2 months to adopt a new minor version).

Does that make sense?

Reply:
Sure, I adapted the document accordingly, please check :)

Reply:
Thanks, I made a few follow-up suggestions (to fit "the last 3 minor versions") below.